tests/console/upload_asset.pm

```diff
@@ -1,27 +0,0 @@
-# SUSE's openQA tests
```

Comment:
I never paid attention to this module, it made me discover the function upload_asset which is actually pretty useful. I wonder if we should keep that module, but schedule it only when we need to get logs, instead of using VNC. But then we'd need to document it. And I think it would be nicer to use a variable in such case, instead of test data.